blockdev_test.go

```diff
@@ -134,7 +134,7 @@ func TestAppendDeviceBlockPFlashRW(t *testing.T) {
 	blkdev := BlockDevice{
 		Driver:    PFlash,
 		ID:        "pflash1",
-		File:      "uefi_nvram.fd",
+		File:      "uefi-nvram.fd",
```

Comment:
Let's leave these as they are; it's an unrelated change to enabling tpm on arm support